loop.sh

```diff
@@ -30,6 +30,10 @@ file_env 'MYSQL_CONNECTION_PARAMS'
 file_env 'MYSQL_HOST'
 file_env 'MYSQL_USER'
 
+if [ -f "$MYSQL_ENV_MYSQL_ROOT_PASSWORD_FILE" ]; then
```

Comment:
why ? it's already done by https://github.com/dsteinkopf/backup-all-mysql/pull/15/files#diff-35a96b9f0b8e5eca6ebf02f011573135a9075ac9c2a961932b6983bb408c1ab8L27 ?